content/html/concepts/semantic-html/terms/section/section.md

```diff
@@ -34,7 +35,96 @@ In the following code, we created a `<section>` element to encapsulate the code.
   </article>
 </section>
 ```
-
+  
+There are also examples where we can use other elements that semantically fit better then `<section>` element.
```

Comment:
These examples belong better in their own example section after the existing example section, rather than in the syntax section. Perhaps with the header "Alternatives to `<section>`"